Torso and Gimbal bringup

younesr1 opened the pull request.

younesr1 commented:
This PR adds changes required to get the torso and gimbal moving. vids of torso and gimbal moving are posted to the discord. moveit_python is removed from movo_third_party and added to the rosinstall instead

nightingale-project committed:
Various fixes for getting the torso and gimbal moving

FelixKHWong commented on movo_common/movo_ros/src/movo_jtas/movo_jtas.py:
```diff
@@ -34,7 +34,7 @@
 --------------------------------------------------------------------"""
 from movo_joint_interface.jaco_joint_controller import SIArmController
-from trajectory_smoother import TrajectorySmoother
+from .trajectory_smoother import TrajectorySmoother
```
should this not be a file path?

younesr1 replied:
both work in pyhthon3

FelixKHWong reviewed (commented):
Seems good since just python fixes. However, why were so many files deleted here?? Did moveit get moved somewhere else or something?

FelixKHWong reviewed (dismissed):
Just wondering but how do you know to delete these files?

younesr1 commented:
> Seems good since just python fixes. However, why were so many files deleted here?? Did moveit get moved somewhere else or something?

i deleted moveit_python and added it as a rosinstall dependency. see movo.rosinstall

younesr1 commented:
just rebased. can i get reapprovals
